File: commands/remember.py
from result import Result


def handle(ctx, user_input: str) -> Result:
    value = user_input.removeprefix("/remember").strip()
    if not value:
        print("Que dois-je me souvenir ?")
        value = input("› ").strip()
    if not value:
        return Result.error("❌ Rien à mémoriser.")
    ctx.memory = value
    ctx.save()
    return Result.success(f"Ok, je me souviens : « {value} »")

File: src/commands/remember.rs
// commands/remember.rs

///Mémoriser une information

use crate::core::input::get_input;
use crate::core::structures::Context;

// fonction handle
pub fn handle_remember(ctx: &mut Context) {
    println!("Que dois-je me souvenir ?");
    ctx.memory = get_input();
    println!("Ok, je m'en souviendrai !");
}